st2common/st2common/models/api/action.py

```diff
@@ -497,6 +497,27 @@ class ActionAliasAPI(BaseAPI, APIUIDMixin):
                 "type": "array",
                 "items": {"type": "string"},
                 "description": "Possible parameter format."
+            },
+            "aliases": {
```

Comment:
Have you considered something along the lines of -
```
formats:
    - display:  "what's next?"
      representation: 
          - "what(.s| is) next\\??"
    - display: "when is {{ query }}?"
      representation: 
           - "when is {{ query }}\\??"
```
This would be horrible to write and also backwards incompatible(we do need to worry about such things soon)

The 2 things that does is -
1. provides a concrete relation between formats and representation.
2. clarifies that is the displayable version and the actual matched version.